.env.example

```
@@ -1,3 +1,6 @@
+POSTGRES_DB="marvticle"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)